snmp/tests/compose/data/netapp.snmprec

```diff
@@ -60,7 +60,7 @@
 1.3.6.1.4.1.789.1.2.1.14.1.14.105.101.102.105.108.101.114.48.51.45.48.49|70|13372368618
 1.3.6.1.4.1.789.1.2.1.14.1.14.105.101.102.105.108.101.114.48.51.45.48.50|70|13957447734
 1.3.6.1.4.1.789.1.2.2.23.0|2|60
-1.3.6.1.4.1.789.1.2.3.8|2|4
+1.3.6.1.4.1.789.1.2.3.8.0|2|4
```

Comment:
Missing .0 for this OID:

> 8: cfInterconnectStatus.0 (INTEGER) down(2)

https://community.netapp.com/t5/ONTAP-Discussions/snmp-takeover-snmpwalk-not-working/m-p/48533